cljfmt/src/cljfmt/core.cljc

```diff
@@ -179,8 +179,13 @@
 (defn- coll-indent [zloc]
   (-> zloc zip/leftmost margin))
 
+(defn uneval-node? [zloc]
```

Comment:
This should be `uneval?` to match other similar functions, it should be private, and it should use the tag to determine the node type rather than converting it to a string:

```clojure
(defn- uneval? [zloc]
  (= (z/tag zloc) :uneval))
```